### PR TITLE
Always minify when encryption is used

### PR DESCRIPTION
**Pull request by BlackBooth**
As suggested, the output ist minified when encryption is used, since nobody can read the output anyways. Test is included

### CODE REVIEW

**Review by ttu — CHANGES_REQUESTED** [review]
Thanks for the PR. Just one minor change request.

File: JsonFlatFileDataStore/DataStore.cs [hunk 1]
```diff
@@ -65,6 +65,7 @@ public DataStore(string path, bool useLowerCamelCase = true, string keyProperty
             }
             else
             {
+                usedFormatting = Formatting.None;
```
[the remark this answers is not in the document]
Comment (BlackBooth) [reply JsonFlatFileDataStore/DataStore.cs]: Good idea, but then I also swapped the logic for the `if (!useEncryption)` to `if (useEncryption)` for better readability (no need to invert the if-clause)

[the remark this answers is not in the document]
Comment (ttu) [reply JsonFlatFileDataStore/DataStore.cs]: Thanks! Yes, it is always to good use positive logic in condition checks 👍

**Review by ttu — APPROVED** [review]

**Comment by BlackBooth** [comment]
Will you merge it?

**Comment by ttu** [comment]
Thanks! Added an update to changelog and merged 👍

**Pull request merged**